ckanext/subscribe/templates/package/snippets/info.html

```diff
@@ -23,6 +23,11 @@
       <form method='post' action="{{ h.url_for('/subscribe/signup') }}" id="subscribe-form" enctype="multipart/form-data" class="form-inline">
         <!-- (Bootstrap 3) <div class="form-group input-group-sm"> -->
           <input id="subscribe-email" type="email" name="email" class="form-control input-small" value="" placeholder="" />
+          {% if apply_recaptcha %}
```

Comment:
Where are we getting this variable from? (I can see that it gets passed in if we render `package/info.html` with `SubscribeController.apply_recaptcha()`, but I don't think we are actually using that controller method anywhere.)